app/javascript/vue/my_module/assigned_items/renderers/name.vue

```diff
@@ -47,6 +48,20 @@ export default {
       axios.get(this.params.data.rowRemindersUrl)
         .then((response) => {
           this.reminders = response.data.html;
+          this.$nextTick(() => {
+            this.$refs.reminders.querySelector('.row-reminders-footer').addEventListener('click', (e) => {
+              let element = e.target;
+              if (e.target.classList.contains('sn-icon')) {
+                element = e.target.parentElement;
+              }
```

Comment:
Wouldn't the even bubble up anyway? I think you just need to use .currentTarget or something like that?